### ISSUE TITLE
need support msys2/mingw

### ISSUE THREAD

**Issue by loohoo** [JBakamovic/yavide#33]
Is yavide can install on msys2/mingw ?

**Comment by JBakamovic** [comment]
This is currently not supported. Unfortunately, I don't have that much experience with `mingw` environment but I am quite sure that installation of `yavide` in such an environment will not work out of the box. There are many dependencies that need to be resolved. 

If you are willing, you can give it try and report the problems you encountered back to me. Maybe we can end up with some solution together ? :)

**Comment by loohoo** [comment]
does it must be Gnome version of Vim?

**Comment by JBakamovic** [comment]
Yes, it shall be an instance of `gVim` with a version no less than 7.3. This is the version where `conceal` feature has been introduced and which is required by some of the plugins used in `yavide`.
